analyzers/rspec/cs/S4581_c#.html

```diff
@@ -10,6 +10,8 @@ <h2>Noncompliant Code Example</h2>
 public void Foo()
 {
     var g = new Guid(); // Noncompliant - what's the intent?
+    var guid = default(Guid);
+    Guid other = default;
```

Comment:
We don't support this yet. Needs to be reverted.

Reply:
Nice catch!